src/mds/MDSRank.cc

```diff
@@ -3579,6 +3584,32 @@ void MDSRank::command_cache_drop(uint64_t timeout, Formatter *f, Context *on_fin
   request->send();
 }
 
+int MDSRank::command_dump_rstats(Formatter *f, std::ostream &ss)
+{
+  if (get_nodeid() != 0) {
+    ss << "I am not mds.0";
+    return -EINVAL;
+  }
+
+  CInode *root = mdcache->get_root();
```

Comment:
Why not have this command accept any path?

Reply:
because mds only tracks if rstats are propagated to root.